src/components/Avatar/Avatar.tsx

```diff
@@ -0,0 +1,49 @@
+import { cva, VariantProps } from "class-variance-authority";
+import { HTMLAttributes } from "react";
+import { cn } from "../../utilities/index";
+import React from "react";
+
+const avatar = cva("inline-flex items-center justify-center font-semibold", {
```

Comment:
I think there's also a Radix Avatar Primitive, that handles fallback and stuff [Radix](https://ui.shadcn.com/docs/components/avatar), or [Shadcn](https://ui.shadcn.com/docs/components/avatar) from which I took some of the components and uses radix under the hood. See in the [repo](https://github.com/zenml-io/zenml-cloud-ui/blob/main/src/components/Avatar.tsx)